t/t0017-security.t

```diff
@@ -137,7 +137,7 @@ test_expect_success 'flux logger not allowed for non-owner' '
 
 test_expect_success 'flux dmesg not allowed for non-owner' '
 	! FLUX_HANDLE_ROLEMASK=0x2 flux dmesg 2>dmesg.err &&
-	grep -q "Operation not permitted" dmesg.err
+	grep -q "Request requires owner credentials" dmesg.err
```

Comment:
Looks like this change snuck into the wrong commit?

Reply:
No that one belongs in the "simplify code" commit because I started printing the error with `log_msg_exit ("...%s", future_strerror (f, errno))` instead of just `log_err_exit ("...")`. 

So the output switched from the canned errno string to the Flux human readable message.

Which is how I noticed it was misspelled :-)

Reply:
Ah! got it, sorry about that!